Source/Scene/Expression.js

```diff
@@ -1198,7 +1198,7 @@ define([
         //>>includeStart('debug', pragmas.debug);
         throw new DeveloperError('Operator "-" requires a vector or number argument. Argument is ' + left + '.');
```

Comment:
They could be `RuntimeError`. I guess my usual thought process is that "the developer wrote a bad style, so throw a developer error", but maybe I need to rethink that.

Returning `-left` is a remnant from the time when the styling language was less type strict. It would let a style like `-"5"` actually become `-5`. I can fix that throughout now that the rules have changed.